doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -3,106 +3,215 @@
 
 This section lists known issues with {PRODUCT} {PRODUCT_VERSION}.
 
-== {CENTRAL}
-
-.Cannot close the standalone editor [https://issues.jboss.org/browse/RHPAM-976[RHPAM-976]]
+== Installer
 
-Issue: If you make a change in a standalone editor, you cannot close the editor until you save your changes.
+.When installing {KIE_SERVER} and {CENTRAL} using the {PRODUCT} installer, the default placeholder for the controller URL should contain {URL_COMPONENT_CENTRAL} [https://issues.jboss.org/browse/RHPAM-1774[RHPAM-1774]]
+Issue: If you use the {PRODUCT} installer to install both {CENTRAL} and {KIE_SERVER}, on the Advanced Configuration page the default value for {KIE_SERVER} is incorrect.
```

Comment:
I assume "on the **Advanced Configuration** page". Comes up a few times, so search all.